app/query_param.go

```diff
@@ -21,7 +24,13 @@ func ParseHistoryQueryParams(c echo.Context, defaultLimit int64) (string, int64,
 		from = time.Now().Unix()
 	}
 
-	return userID, from, limit
+	if err != nil {
+		// If it returns error, it will assume the default behavior(e.g. isBlocked=false).
+		log.Warningln(err)
```

Comment:
Hmm... I made a mistake here May, I thought that we are using logrus, but in fact is a specific logger, this one: https://github.com/topfreegames/mqtt-history/blob/master/logger/logger.go

Can you rewrite this part to use this logger? You can use this code as an example: https://github.com/topfreegames/mqtt-history/blob/master/mongoclient/get_messages.go#L73